### PR TITLE
Enforce quoting style in Travis.

### PR DESCRIPTION
**Pull request by justinwyang**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ matrix:
         - sphinx-build -W -b html -d _build/doctrees source _build/html
         - cd ..
         # Run Python linting, ignore dict vs {} (C408), others are defaults
-        - flake8 --exclude=python/ray/core/generated/,doc/source/conf.py,python/ray/cloudpickle/ --ignore=C408,E121,E123,E126,E226,E24,E704,W503,W504,W605
+        - flake8 --inline-quotes '"' --exclude=python/ray/core/generated/,doc/source/conf.py,python/ray/cloudpickle/ --ignore=C408,E121,E123,E126,E226,E24,E704,W503,W504,W605
```
Comment (ericl) [inline .travis.yml]: Should this be inside the format.sh script?

Comment (robertnishihara) [reply .travis.yml]: Yes, good call. @justinwyang please add this to `format.sh` as well.

Comment (justinwyang) [reply .travis.yml]: @ericl The format.sh file has two calls to flake8, is this to be applied to both calls or just the first one?

Comment (ericl) [reply .travis.yml]: Probably both, unless we want to exclude some files for some reason.

Comment (raulchen) [reply .travis.yml]: Shall we add a project-wide flake8 config file? then we don't need to worry about  inconsistent configs in different scripts.

**Comment by ericl** [comment]
That's fine, but we should stick to using one formatting script. The flake8
call here should be removed.

**Review by robertnishihara — APPROVED** [review]

**Comment by robertnishihara** [comment]
Nice work @justinwyang!

**Comment by jovany-wang** [comment]
I think there is a linting error introduced from this PR.

**Comment by justinwyang** [comment]
I just noticed today, that quotes within a string (ie those written with a backslash) must be single quotes instead of double quotes. I'm not sure how this wasn't detected by this PR's travis, though I am having it fixed within another PR I am working on.

**Comment by robertnishihara** [comment]
@jovany-wang @justinwyang I think the issue was that a new release of `flake8-quotes` was done just a couple days ago. https://pypi.org/project/flake8-quotes/#history

**Comment by robertnishihara** [comment]
As a temporary fix, I'd suggest pinning `flake8-quotes` to 1.0.0 in Travis (or just fixing the linting errors that it's complaining about).

**Pull request merged**